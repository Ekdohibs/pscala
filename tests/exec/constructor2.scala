class A {
  var u = print("new A\n")
}
class B extends A {
  var v = print("new B\n")
}
object Main {
  def main(args: Array[String]) {
    var a = new A();
    var b = new B();
    var c: A = new B()
  }
}
