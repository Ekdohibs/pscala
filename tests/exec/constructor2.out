new A
new A
new B
new A
new B
